pnpm-lock.yaml

```diff
@@ -1,1606 +1,3282 @@
-lockfileVersion: '6.0'
+lockfileVersion: '9.0'
```

Comment:
Which version of `pnpm` does this repo require? I'm on a newer one, so this lockfile got pretty significantly changed here.

Reply:
Currently, there is no `packageManager` field, so there is no predefined version to use. When you used a newer version of `pnpm`, it probably added a `packageManager` field to the package.json

Reply:
Yup, I was hoping to be told what version is preferred so I can just go with that - instead of changing a bunch of things.